requirements.txt

```diff
@@ -3,25 +3,47 @@ cffi==1.1.2
 cryptacular==1.4.1
 cryptography==0.9.1
 enum34==1.0.4
+extras==0.0.3
+fuzzywuzzy==0.5.0
+glob2==0.4.1
 idna==2.0
 ipaddress==1.0.7
+ipython==3.2.0
 Jinja2==2.7.3
+lettuce==0.2.20
+linecache2==1.0.0
+Mako==1.0.1
+Markdown==2.6.2
+markdown2==2.3.0
 MarkupSafe==0.23
+misaka==1.0.2
```

Comment:
I had to look this library up, and it appears to be a markdown rendering library that was last updated about three years ago.  I suspect that some of the other libraries I see in this list are also related and probably should not be used.  One of the tough aspects of using external libraries is the burden of tracking what you are using and what you are not using.  But you need to do this and then clean out packages that are not actually "required" from your `requirements.txt` file.